eventcontent.go

```diff
@@ -35,6 +35,8 @@ type createContent struct {
 	Federate *bool `json:"m.federate"`
 	// The creator of the room tells us what the default power levels are.
 	Creator string `json:"creator"`
+	// The version of the room. Should be treated as "1" when the key doesn't exist.
+	RoomVersion *string `json:"room_version"`
```

Comment:
```suggestion
	RoomVersion *string `json:"room_version,omitempty"`
```

Reply:
Just a note that the `m.federate` key above is also optional. But `createContent` is not a public struct and has limited use so this should not be a big deal.

Reply:
I see, I suppose anything we send back to the client should have `"1"` in there anyways.